org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4graph/GraphViewer.java

```diff
@@ -695,6 +697,16 @@ public Dimension getPreferredSize() {
         return new Dimension((int) (graph.getTotalWidth() * scale), (int) (graph.getTotalHeight() * scale));
     }
 
+    // [HASLab]
```

Comment:
for example:
```bash
find -name "*.java" -print0 | xargs -0 sed -i 's!// \[HASLab\].*!!g'
```
(not tested!)

Reply:
Do we really need the @modified? I find it clutters the code and has no purpose whatsoever? The program is the work of tens of people and if I really want to know who did what Git is there to provide that info more accurately.

Reply:
I agree. This was originally to keep track of changes internally on our side, and then I left them for the PR because I felt it could be helpful for a quick code inspection outside git, but I'll gladly remove them.